tests/main/nfs-support/task.yaml

```diff
@@ -183,11 +183,11 @@ execute: |
     # Ensure that this removed the extra permissions.
     ensure_normal_perms
 
-    # Skip udp protocol on arch-linux and debian-sid because it is not supported
-    # Error displayed:
+    # Skip udp protocol on arch-linux, debian-sid and ubuntu-20.10 because it is
+    # not supported. Error displayed:
     # - arch: mount.nfs: requested NFS version or transport protocol is not supported
-    # - debian-sid: mount.nfs: an incorrect mount option was specified
-    if [[ "$SPREAD_SYSTEM" != arch-* && "$SPREAD_SYSTEM" != debian-sid-* ]]; then
+    # - debian-sid, ubuntu-20.10: mount.nfs: an incorrect mount option was specified
+    if [[ "$SPREAD_SYSTEM" != arch-* && "$SPREAD_SYSTEM" != debian-sid-* && "$SPREAD_SYSTEM" != ubuntu-20.10-* ]]; then
```

Comment:
This seems to be a one way street. We should list the old distributions here, more than exclude the new.

Reply:
Good point, you're right. I'm inclined to leave it as is this time if tests pass and change it the way you suggested on next occasion.